spec.emu

```diff
@@ -75,5 +75,17 @@ contributors: Chengzhong Wu, Ecma International
       </emu-clause>
 
     </emu-clause>
+    <emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+    <emu-clause id="sec-properties-of-the-error-prototype-object">
+      <h1>Properties of the Error Prototype Object</h1>
+        <emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+        <emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+        <emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+        <emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
```

Comment:
About the numbering, I've noticed that the ordering in the spec is alphabetical. Maybe we should follow the ordering too. So preferably the number will be 19.5.3.1 .


![image](https://user-images.githubusercontent.com/8500303/94844275-980fd000-0450-11eb-9eae-12dfec2f638f.png)

Reply:
I felt the same! Shall I change the ordering?